src/modules/addTodo.js

```diff
@@ -1,47 +1,44 @@
 import Task from './Task';
+import '../style.css';
 
 const input = document.getElementById('add-todo');
 const todosUL = document.getElementById('todos');
-const checkbox = document.querySelectorAll('.checkbox');
-const todoEl = document.querySelector('.todo-el');
-// const trash = document.getElementsByClassName('fa-trash-alt');
+const trash = document.getElementsByClassName('positioner');
+const checkBox = document.querySelectorAll('.checkbox');
 
 const todo = new Task();
 
-export function addTodo() {
+export default function addTodo() {
   const todoText = input.value;
   const todoEl = document.createElement('li');
   todo.description = todoText;
-  todo.completed = checkbox.value;
+  todo.completed = checkBox.value;
   todoEl.setAttribute('id', todosUL.childNodes.length);
   todoEl.setAttribute('class', 'todo-el');
   todoEl.innerHTML = `
     <div>
-    <input type="checkbox" tabindex="0" class="checkbox" value="${todo.completed}"/>
-    <label for="checkbox"><span class="todo-text">${todo.description}</span></label>
+    <input 
+    type="checkbox" 
+    tabindex="0" 
+    class="checkbox" 
+    value="${todo.completed}"/>
+    <label for="checkbox">
+    <span class="todo-text">${todo.description}</span>
+    </label>
     </div>
-    <button class="positioner" >
+    <button type="reset" class="positioner" >
     <i tabindex="0" class="fa fa-trash-alt"></i>
     </button> `;
   todosUL.appendChild(todoEl);
-  const todos = [{}];
-  localStorage.setItem(todos, (JSON.stringify([`${todoEl.id}`, `${todo.description}`, `${checkbox.value}`])));
   input.value = '';
+  trash.attachEvent('reset', (e) => {
+    e.preventDefault();
+    todoEl.innerHTML = '';
+    localStorage.clear();
+  });
+  const todos = {};
+  localStorage.setItem(todos, JSON.stringify(todo));
 }
-
```

Comment:
- [ ] As it is requirement for this project, Implement a function for the `Clear all completed` button (use filter() method).